src/types/Server.ts

```diff
@@ -261,6 +261,7 @@ export enum ServerAction {
   UNKNOWN_USER_BADGED = 'UnknownUserBadged',
   TRANSACTION_STARTED = 'TransactionStarted',
   VERIFICATION_EMAIL = 'VerificationEmail',
+  VERIFICATION_EMAIL_USER_IMPORT = 'VerificationEmailUserImoprt',
```

Comment:
Typo: VerificationEmailUserImoprt
We should have used the new enpoint, right?

Reply:
modified